### PR TITLE
Course author documentation for self-paced vs instructor-led

### PR DESCRIPTION
**Pull request by srpearce**
### Date Needed

4 November
### Reviewers
- [ ] Subject matter expert: @marcotuts
- [x] Subject matter expert: @clintonb  
- [ ] Doc team review: @mhoeber or @lamagnifica or @catong 
### Testing
- [ ] Ran ./run_tests.sh without warnings or errors
### Post-review
- [ ] Squash commits

### CODE REVIEW

**Comment by clintonb** [comment]
:+1:

File: en_us/shared/building_and_running_chapters/building_course/setting_pacing.rst [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+course schedule or allow learners to work at their own pace. Courses that
+follow a typical schedule are known as instructor-led courses. Courses that
+allow learners to submit assignments at any time before the course ends,
+without internal due dates, are known as self-paced courses.
+
+.. note::
+ You cannot change the course pacing after the course start date has passed.
+
+**************************************
+Instructor-Led and Self-Paced Courses
```
Comment (marcotuts) [inline en_us/shared/building_and_running_chapters/building_course/setting_pacing.rst]: I will check in tomorrow with Michele about whether "Instructor Led" or "Instructor Paced" is a clearer / more common term. Also happy to lean either way if you have a specific suggestion here @srpearce.

**Comment by marcotuts** [comment]
:+1: 

(One open question about labeling / name of Instructor Led option, but it shouldn't block merge of this, as the current feature uses the name used in the documentation. Thanks! )

**Comment by srpearce** [comment]
Notes from conversation with @marcotuts:

Currently defined MS1 changes are going out today, but will not be enabled. Doc will not be released.

We're going to add two things to MS1 and release them next week: 
- Change "instructor-led" to "instructor-paced"
- Add a field next to the course date in Studio that indicates pacing

These two additional changes will be go out with next week's release, and all of the MS1 changes will be enabled. Doc will release at that time.

**Comment by srpearce** [comment]
@clintonb and @marcotuts, can you review again? I've added student documentation.

Also, @marcotuts, I saw that this is on stage. Is it going to be enabled as well? 

@mhoeber @lamagnifica @catong This should go out in the release on Monday. Can you review it ASAP, please?

**Comment by clintonb** [comment]
:+1:

**Comment by marcotuts** [comment]
:+1:

**Pull request merged**